scripts/release/chebi_update/run_ChEBI_Update.pl

```diff
@@ -0,0 +1,75 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $data_release_pipeline_application_version = "0.0.1-SNAPSHOT";
+my $data_release_pipeline_application = "chebi-update";
+my $data_release_pipeline_tag = "";
+my $data_release_pipeline_repository = "https://github.com/reactome/data-release-pipeline";
+my $resource_dir = "data-release-pipeline/".$data_release_pipeline_application."/src/main/resources/";
+my $start_directory = cwd(); # abs_path($0);
+
+print "start_directory: ".$start_directory;
+
+if (-e "data-release-pipeline") {
+  $logger->info("data-release-pipeline exists, pulling");
+  chdir "data-release-pipeline";
+  system("git pull");
+  # if there is a tag defined, we should check that out.
+  if ($data_release_pipeline_tag ne '')
+  {
+    #chdir "data-release-pipeline";
+    system("git checkout ".$data_release_pipeline_tag);
```

Comment:
I'm confused by this tag handling, why is it assumed the tag would be used with git checkout?

Reply:
In the future we might want to use tags to track versions of the data release pipeline. For example, data-release-pipeline might have version 1.0.3 released and fully tested, but there might be further code on the develop or master branch that we don't want to run in production (for some reason). So, we check out the tag 1.0.3 and then we have _exactly_ the right code. 

Really, this is so we can choose to run a specific version of the code, rather than run whatever is currently on develop.

I plan to actually create a 1.0.0 tag on develop and then update all POMs to have version = 1.0.0 very soon, hopefully before I need to run chebi-update.